src/python/deepgnn/graph_engine/snark/convert.py

```diff
@@ -39,6 +39,8 @@ def __init__(
         skip_edge_sampler: bool = False,
         file_iterator: Optional[TextFileIterator] = None,
         debug: bool = False,
+        is_temporal: bool = False,
+        watermark: int = -1,
```

Comment:
Can we merge these two parameters? e.g. if watermark != -1 then is_temporal is true.

Reply:
Merged